### PR TITLE
Added translation of code-of-conduct to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by fike**
I would like to use this translation to Docker Sao Paulo meetup as code of conduct. But, I think this is could be useful to any meetups to speak in portuguese.

### CODE REVIEW

File: codigo-de-conduta.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+##Introdução
+
+Nós estamos dedicados a prover uma experiência livre de assédio para todos e não toleramos assédio dos participantes em qualquer forma. Nós pedimos a você que seja atencioso com os outros, se comporte profissionalmente e respeitosamente à todos outros participantes. Este código e procedimento relacionados também são aplicados para comportamento inaceitáveis que ocorram fora do escopo das atividades da comunidade, em todos locais da comunidade online e pessoalmente, bem como em toda comunicação individual e onde tal comportamento tenha potencial de prejudicar a segurança e o bem estar dos membros da comunidade. Exibidores, palestrantes, patrocinadores, staff e todos os outros participantes em eventos organizados pela Docker, Inc (DockerCon, meetups, grupo de usuários) ou mantidos em instalações Docker Inc. estão sujeitos a estas Diretrizes e Código de Conduta da Comunidade.
+
+Diversidade e inclusão faz comunidade Docker forte. Nós encorajamos a participação a maior variedade e diversidade possível, queremos ser bem claro onde estamos.
```
Comment (joaofnfernandes) [inline codigo-de-conduta.md]: Diversidade e inclusão tornam a comunidade Docker forte

File: codigo-de-conduta.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Nosso objetivo é manter uma comunidade Docker amigável, útil e segura para todos, independente da experiência, identidade e expressão de gênero, orientação sexual, deficiência, aparência pessoal, tamanho do corpo, raça, etnia, idade, religião, nacionalidade ou qualquer categoria protegida nos termos da lei.
```
Comment (joaofnfernandes) [inline codigo-de-conduta.md]: Em vez de "tamanho do corpo", que tal "aparência"?

File: codigo-de-conduta.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+##Expectativa de comportamento
+
+- Ser profissional.
+- Ser responsável.
+- Ser receptivo.
+- Ser gentil.
+- Ser respeitoso ao outros pontos de vista e ideias.
```
Comment (joaofnfernandes) [inline codigo-de-conduta.md]: Que tal "Respeitar pontos the vista e ideas variadas"

File: codigo-de-conduta.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- Ser solidário e respeitoso um com outro.
```
Comment (joaofnfernandes) [inline codigo-de-conduta.md]: Que tal "Ser solidário e olhar pelos outros"

File: codigo-de-conduta.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+##Comportamento não esperado
+
+Assédio pode incluir a lista abaixo mas não limitado à ela:
+
+- Comentários ofensivos, inadequados ou indesejáveis ao gênero, orientação sexual, deficiência, aparência física, tamanho do corpo, raça, etnia, nacionalidade, religião, idade ou qualquer outra categoria protegida pela lei.
+- Assédio visual, ex: imagens sexual ou uso de linguagem sexual nos eventos da comunidade Docker
```
Comment (joaofnfernandes) [inline codigo-de-conduta.md]: ... imagens sexuais ....

File: codigo-de-conduta.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- Desrespeito às diferentes opiniões
+- Intimidação deliberada, perseguição, gravação ou fotografia constrangedoras
+- Interrupção continua das palestras ou outros eventos
+- Contato físico inapropriado ou indesejado
+- Intimidação ou bullying (online ou pessoalmente)
+- Atenção sexual não desejada
+
+##Relatórios e Aplicação
+
+- Se você sofrer ou testemunhar qualquer violação do Código de Conduta, por favor, entre me contato conosco pelo relatório de incidente (inglês) [incident report](https://docs.google.com/forms/d/e/1FAIpQLScezna1ZXRPzC_phSDoPEF4c5nvw8yQW-vvtI8xHjv-BB9MOg/viewform?c=0&w=1), ou poremail dockercon@docker.com
```
Comment (joaofnfernandes) [inline codigo-de-conduta.md]: ... entre em contato conosco através do relatório de incidentes (inglês) ... ou por email ...

**Comment by sanscontext** [comment]
Thanks @joaofnfernandes - I think you're right. @fike would you rename this so the filename is parallel to the original?  And maybe list it in the README so people have more of a hint that it exists. (I'm excited to have more translations! :D )

**Comment by fike** [comment]
For sure! Thansk @sanscontext and @joaofnfernandes. I'll do. :)

File: codigo-de-conduta.md (hunk 2 above)
Comment (fike) [inline codigo-de-conduta.md]: Blz. A

File: codigo-de-conduta.md (hunk 3 above)
Comment (fike) [inline codigo-de-conduta.md]: Boa, talvez fosse melhor. "Respeitar o ponto de vista dos outros e ideais diferentes". O que acha? @joaofnfernandes ?

File: codigo-de-conduta.md (hunk 4 above)
Comment (fike) [inline codigo-de-conduta.md]: Boa também, poderia ser então "Ser solidário e respeitar o próximo". @joaofnfernandes

File: codigo-de-conduta.md (hunk 6 above)
Comment (fike) [inline codigo-de-conduta.md]: ok

**Comment by sanscontext** [comment]
Looks good, and confirmed with @joaofnfernandes offline that it gets the 👍 .  Merging, and I'll add a line to our README listing it, for better visibility's sake. :)  

Thanks again!

**Pull request merged**